test/client/fixtures/sandbox/node/classes-test.js

```diff
@@ -173,6 +173,23 @@ if (window.MutationObserver) {
     });
 }
 
+module('PerformanceEntry');
```

Comment:
the `module` directive needs for group a lot of tests.
I see here only one.
Remove the `module` directive and renamte test as `PerformanceNavigationTiming.name`.